CodeLanguages-Container/CodeLanguages-Container.xcodeproj/project.pbxproj

```diff
@@ -670,6 +674,14 @@
 				kind = branch;
 			};
 		};
+		6CAB9CA029C2CE0E006B8521 /* XCRemoteSwiftPackageReference "tree-sitter-regex" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/thecoolwinter/tree-sitter-regex";
```

Comment:
Please make sure to create a `feature-spm` branch on your fork so everyone knows, that this is not the original repo.

Reply:
Done 👍